app/javascript/src/components/Dashboard/Contacts/constants.js

```diff
@@ -82,3 +83,17 @@ export const ROWS = [
 ];
 
 export const CONTACTS_ROWS = Array(30).fill(ROWS).flat();
+
+export const CONTACTS_FORM_INITIAL_FORM_VALUES = {
+  first_name: "",
+  last_name: "",
+  email: "",
+  address: "",
+};
+
+export const CONTACTS_FORM_VALIDATION_SCHEMA = yup.object().shape({
+  first_name: yup.string().required("First name is required"),
+  last_name: yup.string().required("Last name is required"),
+  email: yup.string().required("Email is required"),
```

Comment:
Yup has a specific email validator. Use it.

Reply:
Added. Thanks